### ISSUE TITLE
Fix accessibility issues for dropdown in altinn-components

### ISSUE THREAD

**Issue by mbacherycz** [Altinn/dialogporten-frontend#2012]
### Description

Fix accessibility violation for dropdown in altinn-components library. 
Potentially removing aria role "menu" from dropdown will fix the issue. 
After that, we need to also uncomment accessibility tests in AF in frontend/axe.test.ts 

Ref: https://github.com/Altinn/dialogporten-frontend/pull/2007

<img width="1473" alt="Image" src="https://github.com/user-attachments/assets/befebbd0-4a46-4b67-bf82-4486fa9a6c98" />
<img width="839" alt="Image" src="https://github.com/user-attachments/assets/69654246-d6de-4918-8d47-a616f3a1dd20" />

**Comment by mbacherycz** [comment]
This is a duplicate of: 
https://github.com/Altinn/altinn-components/issues/361 

Closes.
